sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs

```diff
@@ -34,7 +34,11 @@ protected ConversationAnalysisClient()
         {
         }
 
-        /// <summary> Analyzes the input conversation utterance. </summary>
+        /// <summary>
+        /// [Protocol Method]Analyzes the input conversation utterance.
```

Comment:
At the very least there should be a space:
```suggestion
        /// [Protocol Method] Analyzes the input conversation utterance.
```
But this looks very unprofessional, IMO. This isn't a GitHub bug title where people "tag" titles (instead of relying a labels). Could we at least make it read like documentation e.g.,

"Protocol method that analyzers the input...".

The generator could to-lowercase the first letter and prepend "Protocol method that " safely, I'd imagine. Whether the first letter is a vowel or not doesn't matter like with some articles.

Reply:
This was the agreed on syntax I can add the space in a follow up PR.